src/helpers.js

```diff
@@ -1,7 +1,8 @@
 import fetch from 'cross-fetch';
 import { defaultAbiCoder } from 'ethers/utils/abi-coder';
 
-export const AGGREGATE_SELECTOR = '0x9af53fc6';
+// Function signature for: aggregate((address,bytes)[])
```

Comment:
woah, never seen a solidity function sig w/ a struct before

Reply:
Yeah same – such a nice and succinct way of representing the data structure :D